index.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-restricted-syntax */
+
+import { addBook, removeBook, populateBooksFromStorage } from './modules/books-manager.js';
+import { $select, $selectById } from './modules/selectors.js';
+
+const anchors = $select('.navbar a', true);
+
+function switchSection(event) {
+  event.preventDefault();
+
+  this.classList.toggle('active', true);
+
+  let recentSectionId;
```

Comment:
- [ ] Please as requested by the previous reviewer, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
As I mentioned above to the previous reviewer, an arrow function doesn't provide reference to the element on which the event occurred, therefore, a regular function is the best bet in this scenario.